astropy/config/tests/test_configs.py

```diff
@@ -353,7 +353,7 @@ class Conf(ConfigNamespace):
     with open(f, "wb") as fd:
         apycfg.write(fd)
     with open(f, encoding="utf-8") as fd:
-        lns = [x.strip() for x in fd.readlines()]
+        lns = fd.read().splitlines()
```

Comment:
_Should_ not matter since it was likely just for removing the linebreaks (see previous discussion); I think other blanks will be stripped by the column parser anyway.

Reply:
@taldcroft sorry we ended up mixing auto formatting with manual refactors. To be clear here's our process

```
[x.strip() for x in fd.readlines()]
⬇️ (ruff)
[x.strip() for x in fd]
⬇️ (manual refactor)
fd.read().splitlines()
```

Reply:
This refactor was moved to #16656. Please consider closing this thread.